R/db.R

```diff
@@ -407,7 +407,7 @@ get_new_key <- function(dbconn, tbl_name, ids, key) {
   } else {
     shards <- shards[[1]]
   }
-  if (!DBI::dbExistsTable(dbconn, shards[1])) return(ids)
```

Comment:
@kirillseva @robertzk , @khalidharun says that right now if the table doesn't exist, it returns `ids` and says that *all* of them are cached.  Instead, @khalidharun thinks we should return nothing to show that *none* of them are cached.  Can you verify this?